src/models/data_models/IConfiguration.ts

```diff
@@ -0,0 +1,31 @@
+export interface IConfiguration {
+	googlePlaces: GooglePlaces;
+	yelp: Yelp;
+	openCage: OpenCage;
+}
+
+export interface Yelp {
```

Comment:
```suggestion
export interface YelpConfig {
```

likewise for the other interfaces